x/wasm/module_adapter.go

```diff
@@ -33,18 +33,11 @@ func (b AppModuleBasic) RegisterCodec(amino *codec.Codec) {
 }
 
 func (b AppModuleBasic) DefaultGenesis() json.RawMessage {
-	return ModuleCdc.MustMarshalJSON(&GenesisState{
-		Params: DefaultParams(),
-	})
+	return nil
 }
 
 func (b AppModuleBasic) ValidateGenesis(message json.RawMessage) error {
-	var data GenesisState
-	err := ModuleCdc.UnmarshalJSON(message, &data)
-	if err != nil {
-		return err
-	}
-	return ValidateGenesis(data)
+	return nil
```

Comment:
validate should not be deleted because it can be used in other places

Reply:
I make wasm the same as ibc module. And Validate Genesis is only called by `CMD`. This code delete is for wasm is not supported in genensis file